### ISSUE TITLE
Question: Deacessioned datasets still provide files

### ISSUE THREAD

**Issue by hofmannc** [MPDL/dataverse#19]
Server: prod-edmond2 & dev-edmond2
Date of Test: 07.12.2021
Browser: ff
Version: Dataverse v. 5.8
User: Admin

Action: select dataset -> "edit dataset" -> "Deaccession Dataset" -> "enter reason" -> "deaccession" -> "Are you sure" -> yes "No files in dataset" -> Deaccession notice and reason is displayed"

Observation: Files are still available and downloadable
Question: in Imeji, after withdrawing the collection, only the metadata is still available. Is the dataverse procedure correct?

**Comment by haarli** [comment]
Yes, in dataverse the files are kept when deaccessioning a dataset. However, they are not public anymore, so only Admins and the Dataset creator can  see them. This way, it is even possible to create a new version of deaccessioned dataset (e.g. to fix dataset errors and publish a new error-free version)
